### PR TITLE
Adds taj buckethat sprites

### PR DESCRIPTION
**Pull request by TheStryker**
Adds a bucket hat sprite that doesn't clip for tajara.

image

### CODE REVIEW

**Comment by TheStryker** [comment]
!review

File: html/changelogs/stryker-tajbuckethat.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+author: The Stryker
+delete-after: True
+changes: 
+  - rscadd: "Adds a sprite for the buckethat that won't clip with tajara ears."
```
Comment (fernerr) [inline html/changelogs/stryker-tajbuckethat.yml:4]:
```suggestion
  - imageadd: "Adds a sprite for the buckethat that won't clip with tajara ears."
```

**Review by fernerr — APPROVED** [review]

**Review by Geevies — APPROVED** [review]

**Comment by alsoandanswer** [comment]
This is technically a bugfix since tajara ear clipping is an unintended consequence

**Comment by TheStryker** [comment]
!bugfix

**Pull request merged**